app/src/main/res/values/strings.xml

```diff
@@ -122,6 +121,7 @@
     <string name="first_time">FIRST_TIME</string>
     <string name="ok">OK</string>
     <string name="invalid_email">Invalid Email</string>
+    <string name="email_invalid_title">Incorrect Email</string>
```

Comment:
Suggestion: Avoid these kinds of changes (Shifting of one line from one place to another). They cause conflicts.